brjs-sdk/sdk/libs/javascript/br-i18n/src/br/i18n/Translator.js

```diff
@@ -46,7 +46,7 @@ function Translator(messageDefinitions, useLocale) {
 
 	/** @private */
 	this.localizationPrefs = {};
-	this.setLocale(useLocale);
+	this.setLocale(useLocale || 'en');
```

Comment:
may this cause issues if 'en' is not one of the defined locales for an app, if so couldn't we use 'defaulLocale' which is defined above? or is this for the case when no locales have been defined?